src/Api/ApiClient.php

```diff
@@ -312,4 +328,38 @@ private function decodeData(array $responseData, SignatureDataFormatter $signatu
 		return $responseData;
 	}
 
+	private function logRequest(string $url, array $queries, array $requestData = null, Response $response)
+	{
+		if ($this->logger === null) {
+			return;
+		}
+
+		$context = [
+			'type' => 'csobRequest',
+			'response-code' => $response->getResponseCode()->getValue(),
+			'response-data' => $response->getData(),
+		];
+
+		if (count($queries) !== 0) {
+			$context['queries'] = $queries;
+		}
+
+		if ($requestData !== null) {
+			$context['request-data'] = $requestData;
+		}
+
+		if (count($_GET) !== 0) {
+			$context['GET'] = $_GET;
+		}
+
+		if (count($_GET) !== 0) {
+			$context['POST'] = $_POST;
+		}
+
+		unset($context['request-data']['signature']);
+		unset($context['response-data']['signature']);
```

Comment:
signature se může vyskytovat i v `[response-data][extensions]` (extensions je pole). To by se taky mělo odstranit